File: BTT4/BTT4 (BTT3 UPDATE)/BTT3/LogInGUI.cpp
#include "LogInGUI.h"
#include<iostream>
#include "userInfo.h"
#include "Menu.h"
#include "Controller.h"

using namespace std;
string LogInGUI::InputUserNameandPassword(string userName, string passWord)
{
	Controller Controller_GUI;
	menu menuLog;
	userInfo userInfo_LogInGUI;
	if (Controller_GUI.CheckInputUserNameAndPassWord(userName, passWord) == true) {
		return userInfo_LogInGUI.TakeUserID(userName);
	}
	else {
		cout << "LogIn NO sucsessfully, Input again!!!" << endl;
		menuLog.Displaymenu();
		return 0;
	}

}
LogInGUI::LogInGUI()
{
}


LogInGUI::~LogInGUI()
{
}

File: BTT4_PY1/BTT4_PY1/LogInGUI.py
from Controller import Controller
class LogInGUI():
	def RequestLogIn():
		contr = Controller
		print("moi ban nhap userName: ")
		userName = input()
		print("moi ban nhap passWord: ")
		passWord = input()
		print(contr.RequestLogIn(userName,passWord))
		return contr.RequestLogIn(userName,passWord)


